Web of Science.js

```diff
@@ -0,0 +1,211 @@
+{
+	"translatorID": "88e11bcb-464d-4b6d-a446-8994e3b865c9",
+	"label": "Web of Science",
+	"creator": "Philipp Zumstein",
+	"target": "^https?://[^/]*webofknowledge\\.com/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
```

Comment:
Yes, it require subscription. BTW I just marked everything that is my default setting, until you tell me what doesn't work ;-) How about "gcsib" then?